lola-tech/test/definitions/homepage_definitions.js

```diff
@@ -1,8 +1,10 @@
-import { Given } from "cucumber";
+import { Given, Then } from "cucumber";
 import HomePage from "../pageobjects/homepage.js";
 
-Given(/^I access the URL "([^"]*)"$/, function (url) {
+Given(/^I access the URL "([^"]*)"$/, function(url) {
   HomePage.openURL(url);
 });
 
-
+Then(/^I validate that "([^"]*)" page opens in a new tab$/, function(tab) {
+  HomePage.validateNewTab(tab);
+});
```

Comment:
This should be removed to the generic definitions file, I think. It's not a definition specific to the Homepage.

Reply:
Please also change the name of this method to be a switchToNewTab method.